plotly/tools.py

```diff
@@ -386,23 +386,20 @@ def mpl_to_plotly(fig, resize=False, strip_style=False, verbose=False):
 
 ### graph_objs related tools ###
 
-# TODO: Scale spacing based on number of plots and figure size
-def get_subplots(rows=1, columns=1, horizontal_spacing=0.1,
-                 vertical_spacing=0.15, print_grid=False):
+def get_subplots(rows=1, columns=1, print_grid=False, **kwargs):
     """Return a dictionary instance with the subplots set in 'layout'.
 
     Example 1:
-        # stack two subplots vertically
-        fig = tools.get_subplots(rows=2)
-        fig['data'] += [Scatter(x=[1,2,3], y=[2,1,2], xaxis='x1', yaxis='y1')]
-        fig['data'] += [Scatter(x=[1,2,3], y=[2,1,2], xaxis='x2', yaxis='y2')]
+    # stack two subplots vertically
+    fig = tools.get_subplots(rows=2)
+    fig['data'] += [Scatter(x=[1,2,3], y=[2,1,2], xaxis='x1', yaxis='y1')]
+    fig['data'] += [Scatter(x=[1,2,3], y=[2,1,2], xaxis='x2', yaxis='y2')]
 
```

Comment:
yeah, let's give one example and link to documentation, we can use that thing to link to the appropriate domain too: https://github.com/plotly/python-api/blob/80fcc59309d59c227a91075070882904e29fb006/plotly/utils.py#L286-292

with the `@utils.template_doc` wrapper, like in https://github.com/plotly/python-api/blob/a0dad3cc0b70a60e195d4a6ea7eaca06937d1bf3/plotly/plotly/plotly.py#L391